tech.json

```diff
@@ -450,7 +450,15 @@
                       "name": "canSpringBallBounce",
                       "techRequires": ["canLateralMidAirMorph"],
                       "otherRequires": ["SpringBall"],
-                      "note": "Using a lateral mid-air morph to bounce off a surface with Spring Ball while retaining previous momentum."
+                      "note": "Using a lateral mid-air morph to bounce off a surface with Spring Ball while retaining previous momentum.",
+                      "extensionTechs": [
+                        {
+                          "name": "canTrickySpringBallBounce",
+                          "techRequires": ["canSpringBallBounce"],
+                          "otherRequires": [],
+                          "note": "Performing spring ball bounces in tricky situations, such as squeezing into a Morph tunnel while bouncing to keep blue speed."
```

Comment:
```suggestion
                          "note": "Performing spring ball bounces in tricky situations, such as precisely platforming while bouncing to keep blue speed."
```

Reply:
I see `canTrickySpringBallBounce` as springballing through a room like pre-ridley, gauntlet, or LN-pillars.  The climb supers situation is more about landing the single sinking morph into the hole, but its worded here too closely to the `canTemporaryBlue` bounce into a morph tunnel.  But it still makes sense for it to apply here.